CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Unreleased
 
-- ...
+- Fixed curl verify host for synchronous mode (#767)
+- Use `mb_substr` instead of `substr` if available everywhere (#734)
+- Make it possible to changeable `default_max_depth` in `Raven_Serializer` (#632)
```

Comment:
Change the phrase to ``Make it possible to change the `default_max_depth` option in `Raven_Serializer` ``